### ISSUE TITLE
Finetuning torchvision models with c++ API

### ISSUE THREAD

**Issue by ShahriarSS** [pytorch/pytorch#15275]
## ❓ 
Hi. I was searching for a way to bring torchvision models into c++ and finetune them. I came across this:

https://pytorch.org/tutorials/advanced/cpp_export.html

and I managed to load some of the pretrained models in c++ but my question is that how can I finetune and save the model. I couldn't find anything about this online. torch::jit::script::Module has a forward method but doing backward on the loss from the output of forward doesn't work. What can I do?

**Comment by soumith** [comment]
use https://discuss.pytorch.org for questions
